<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionDialog</class>
 <widget class="QWidget" name="TransactionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>483</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>483</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>483</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.dialog_frame {
		background: #F4F4F4;
	border: none;
}
.frame_header_dialog {
		background: #F4F4F4;
        border-bottom: 1px solid gray;
}
.label_h1, logo {
	background: #F4F4F4;
}

QTextEdit {
        background: white;
        color: black;
}
QDateEdit {
	background: white;
	color: black;
}
Spacer {
	background: #F4F4F4;
}
QLabel {
    color: black;
	background: #F4F4F4;
}
QGroupBox {
    border: 1px solid gray;
    border-radius: 5px;
    padding-top: 10px;
	margin-top: 8px;
}
QGroupBox::title {
	color: black;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px 10px;
}
QComboBox, QAbstractItemView{
	color: black;
	background: white;
}
QComboBox::Items {
	color: black;
}
QLineEdit {
	color: black;
	background: white;
}

.pushButton_action {
        border: 1px solid rgb(170,0,0);
        border-radius: 8px;
        padding: 6px;
        color: rgb(170,0,0);
        font-weight: bold;
        font-size: 12px;
}
.pushButton_action::hover {
        background: rgb(170,0,0);
        color: white;
}
</string>
  </property>
  <property name="class" stdset="0">
   <string>dialog_frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_header">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="class" stdset="0">
      <string>frame_header_dialog</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_logo">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>PennyWise Logo</string>
        </property>
        <property name="statusTip">
         <string>PennyWise Logo</string>
        </property>
        <property name="whatsThis">
         <string>PennyWise Logo</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources/resources.qrc">:/logo/logo</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="class" stdset="0">
         <string>logo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_header">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Transaktion hinzufügen</string>
        </property>
        <property name="class" stdset="0">
         <string>label_h1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_content">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="class" stdset="0">
      <string>dialog_frame</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_data">
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="class" stdset="0">
         <string>dialog_frame</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Daten</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Datensatznr.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbl_id">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Konto:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBox_account">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Datum:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateEdit" name="dateEdit_date">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Empfänger (opt):</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_recipient">
              <property name="maxLength">
               <number>30</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Verwendungszweck (opt):</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QTextEdit" name="textEdit_purpose">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="class" stdset="0">
               <string>textEdit_purpose</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Betrag:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <property name="class" stdset="0">
               <string>dialog_frame</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="lineEdit_amount">
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_currency">
                 <property name="text">
                  <string>EUR</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Transaktionstyp:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="lbl_type">
              <property name="text">
               <string>NaN</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Hauptkategorie (opt):</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QComboBox" name="comboBox_category">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Unterkategorie (opt):</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QComboBox" name="comboBox_sub_category">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_btn">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="class" stdset="0">
         <string>dialog_frame</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_cancel">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Abbrechen</string>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="class" stdset="0">
            <string>pushButton_action</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_submit">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Speichern</string>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="class" stdset="0">
            <string>pushButton_action</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
